### PR TITLE
Add repo docs

### PR DESCRIPTION
**Pull request by davidicus**
These are a few of the docs Taylor and I use in our projects. Could possibly be helpful. @kellychurchill please review and let me know if 👍  or 👎

### CODE REVIEW

**Comment by davidicus** [comment]
I also wanted to ask before adding a License here.

**Review by kellychurchill — APPROVED** [review]
Looks like a good start - thanks!

**Comment by kellychurchill** [comment]
For License for this project specifically, I think we can take the lead from what other starter projects have done https://www.gatsbyjs.org/docs/gatsby-starters/

**Comment by davidicus** [comment]
@kellychurchill The default one has an MIT license. I have added to this PR. If it's cool with you I will merge.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # gatsby-starter-carbon
 Gatsby starter with the Carbon Design System
+
+##License
```
Comment (kellychurchill) [inline README.md]: Formatting is messed up a bit  here - should be a space between the hashtag and License

**Pull request merged**